financial-connections/src/main/java/com/stripe/android/financialconnections/domain/PollAuthorizationSessionAccounts.kt

```diff
@@ -27,66 +30,80 @@ internal class PollAuthorizationSessionAccounts @Inject constructor(
     suspend operator fun invoke(
         canRetry: Boolean,
         manifest: FinancialConnectionsSessionManifest
-    ): PartnerAccountsList {
-        return retryOnException(
-            times = MAX_TRIES,
-            delayMilliseconds = POLLING_TIME_MS,
+    ): PartnerAccountsList = try {
```

Comment:
try now wraps the `retryOnException` execution, so that `toDomainException` also maps retry exceptions.